### ISSUE TITLE
Support zoom in/out in console

### ISSUE THREAD

**Issue by yanyu2015** [spyder-ide/spyder#20123]
how to zoom in/out text in console? Notice that it's not the editor.
![image](https://user-images.githubusercontent.com/11142054/204170825-2ac65116-d305-42bf-9bf9-221e2453826a.png)

**Comment by ccordoba12** [comment]
Hey @yanyu2015, thanks for reporting. You need to use the same shortcuts as in the editor for this, i.e. `Ctrl + +` to increase the text size in the console and `Ctrl + -` to decrease it.

However, since there's no menu entry that shows that, I think we should add it so that this functionality can be easily discovered by users.

**Comment by yanyu2015** [comment]
How about support that `ctrl+mouse scroll up` is equal to `Ctrl + +`, and `ctrl+mouse scroll down` is equal to `Ctrl + -`.

**Comment by ccordoba12** [comment]
Good idea. That will make the support to zoom in/out in the console work the same as in the Editor.

**Comment by Dionisis1979** [comment]
Hello!
Using Spyder 5.5.0 standalone on Windows 11 home 64 bit here.
Somehow, only the shortcut Ctr+- is working on the console, i.e. the zoom out. The shortcut Ctr++ is not working.
It's been a while since I worked on Spyder but I remember it used to work all right earlier on, with the exact same installation I have now. Only thing that has changed meanwhile is some windows updates.
Would a clean Spyder install solve the problem?

**Comment by Yusufbeg** [comment]
Looks like Ctrl++ has a common bug, it doesn't work. Alternatively, press Ctrl+, (MacOS: Cmd+,) or go to Preferences and increase font size in Appearance.

**Comment by ccordoba12** [comment]
@jsbautista, I think this requires changes in QtConsole.
